### PR TITLE
Header component

### PR DESCRIPTION
**Pull request by sanjanadhikari1**
- [ ] install tailwindcss on the project using Cris's code.
- [ ] create pixel perfect header component
- [ ] make it responsive using tailwindcss

FreedCamp
[https://freedcamp.com/view/2956183/tasks/40068433]()

### CODE REVIEW

File: src/common/components/Icon/Icon.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import * as React from 'react'
+import * as Styles from './Icon.styles'
+
+export interface IIconProps {
+  variant?: string
```
Comment (br4in3x) [inline src/common/components/Icon/Icon.tsx:5]: Can we provide all variants here, like `noti | default`, or any others?

File: src/common/constants/routes.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (br4in3x) [inline src/common/constants/routes.json:1]: We will have to change this to look like this: https://github.com/pastelnetwork/pastel-electron-wallet/blob/d48c4a7947e3af23f59990c5f4e706fbfcdde639/src/common/routes/index.tsx

File: src/features/header/HeaderScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+  const inputRef = useRef<HTMLInputElement>(null)
+  let placeholder = 'Search creator or NFT'
+  if (inputRef && inputRef.current) {
+    if (inputRef.current.clientWidth < 200) {
+      placeholder = 'Search creator ...'
+    }
+  }
+  return (
+    <div className='flex relative'>
+      <img width='16' className={styles.searchIconPosition} src={searchIcon} />
```
Comment (br4in3x) [inline src/features/header/HeaderScreen.tsx:68]: We need to use styled components, please

File: src/legacy/Routes.tsx [hunk 4]
```diff
@@ -11,6 +11,7 @@ import Dashboard from './components/Dashboard'
 import Send from './components/Send'
```
Comment (br4in3x) [inline src/legacy/Routes.tsx:11]: We will have to move changes from here to the new Routes file, defined here: https://github.com/pastelnetwork/pastel-electron-wallet/blob/d48c4a7947e3af23f59990c5f4e706fbfcdde639/src/common/routes/Routes.tsx.

**Review by br4in3x — COMMENTED** [review]
This PR should wait until https://github.com/pastelnetwork/pastel-electron-wallet/pull/86/files is merged, because https://github.com/pastelnetwork/pastel-electron-wallet/pull/86/files sets up new routing architecture that would be right for us.

**Comment by CrazyOptimist** [comment]
Closing this now, it's merged into `new_development`.

**Pull request closed without merging**